static/data/hierarchy_statsvar.json

```diff
@@ -3,22 +3,22 @@
   "sv": "top",
   "l": "Demographics",
   "t": "p",
-  "c": 1784,
+  "c": 1785,
   "cd": [
    {
-    "sv": "Count_Person",
+    "sv": "dc/cv35g4gzkth83",
```

Comment:
can you get all the stats var equivalent to Count_Person and prefer the human stats var, same for the rest of the list

Reply:
Added a function to remove duplicates